coalib/bears/Bear.py

```diff
@@ -18,8 +19,26 @@
 from coalib.settings.FunctionMetadata import FunctionMetadata
 from coalib.settings.Section import Section
 from coalib.settings.ConfigurationGathering import get_config_directory
-
```

Comment:
look up PEP8 order of imports ;)

Reply:
Actually, my PR still needs the linting :stuck_out_tongue: , I haven't run `coala` on this change, was waiting for the review